uk.co.agilemde.agileuml.yaml

```diff
@@ -0,0 +1,81 @@
+id: uk.co.agilemde.agileuml
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: agileuml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk21/installjdk.sh
+
+  - name: agileuml
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/eclipse/agileuml/
+        tag: v2.3
+        commit: 4cdbdb96169c15bce63855b099a620ad9d3bbc31
+        x-checker-data:
+          type: git
+          tag-pattern: ^v\\d([.\\d]+)$
+
+      # Added because released asset file umlrsds.jar could be different
+      # to umlrsds.jar in repository at specified tag and commit.
+      - type: file
+        url: https://github.com/eclipse/agileuml/releases/download/v2.3/umlrsds.jar
+        sha256: 34d8a826ad754bd871d7535e778d1beb0426e8279a686ad1b9bef6e5ca08623f
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/eclipse/agileuml/releases/latest
+          version-query: ".tag-name"
+          url-query: ".assets[] | select(.name==\"umlrsds.jar\") | .browser_download_url"
+      
+      - type: file
+        path: uk.co.agilemde.agileuml.appdata.xml
+      
+      - type: file
+        path: uk.co.agilemde.agileuml.desktop
+
+      - type: file
+        path: logo.png # 280x280
+
+      - type: file
+        path: logo32x32.png
+
+      - type: file
+        path: logo64x64.png
+
+      - type: file
+        path: logo128x128.png
+
+      - type: file
+        path: logo256x256.png
+      
+      - type: script
+        dest-filename: agileuml
+        commands:
+          - export APPDATA="$XDG_DATA_HOME"
+          - export PATH="/app/jdk/bin:$PATH"
+          - exec java -jar /app/bin/umlrsds.jar "$@"
+    
+    build-commands:
+      - chmod +x umlrsds.jar
+      - install -d /app/share/agileuml/
+      - cp -r * /app/share/agileuml
+      - install -Dm755 umlrsds.jar /app/bin/umlrsds.jar
+      - install -Dm755 agileuml /app/bin/agileuml
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 $FLATPAK_ID.appdata.xml /app/share/appdata/$FLATPAK_ID.appdata.xml
+      - install -Dm644 logo.png /app/share/icons/$FLATPAK_ID.png
+      - install -Dm644 logo32x32.png /app/share/icons/hicolor/32x32/apps/$FLATPAK_ID.png
+      - install -Dm644 logo64x64.png /app/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+      - install -Dm644 logo128x128.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
```

Comment:
```suggestion
```

Reply:
This commit removes them from the Flatpak and uses the URLs to said files from my PR instead (it has only been opened tonight and hasn't been merged as of yet): https://github.com/flathub/flathub/pull/5329/commits/42dee1ffc50745bc704d076f51a82bf489b07290

Reply:
You don't need the smaller sized icons, supply the largest one